docs/features.md

```diff
@@ -2,11 +2,44 @@
 
 ## Standard Features
 
+### Data Organization
```

Comment:
this section was in setup.md, which was a mistake. i migrated it here and fleshed it out